### ISSUE TITLE
install to vs community mac

### ISSUE THREAD

**Issue by bgbruno** [Microsoft/nodejstools#2093]
Hello MS VS,

I don't it see anywhere

pls how can I install this in "ms vs mac" ?

**Comment by anutting** [comment]
yeah how can I install this on mac? Node.js support is advertised all over the Visual Studio page, but not for mac?

**Comment by KirillOsenkov** [comment]
Hi folks, 

unfortunately right now Visual Studio for Mac does not support NodeJS Tools. If you'd like to see this supported in the future I recommend you open a new idea/suggestion over at: https://developercommunity.visualstudio.com/content/idea/post.html?space=41

**Comment by bgbruno** [comment]
thx for answer @KirillOsenkov good to know that still nothing

also I thought it would be enough point that issue here - I don't

**Comment by amehat** [comment]
Hello,

where is this functionality? It is really important.
